src/Spark/Core/Handler/StatusHandler.cls

```diff
@@ -110,7 +110,7 @@ class Spark.Core.Handler.StatusHandler inherits OpenEdge.Web.WebHandler use-widg
                 oWriter:write('</table>').
                 oWriter:write('</section>').
 
-                oWriter:write('<footer>Copyright &copy; 2020 Progress Software Corporation. All rights reserved.</footer>').
+                oWriter:write('<footer>Copyright &copy; ' + string(year(today)) + ' Progress Software Corporation. All rights reserved.</footer>').
```

Comment:
Is this correct? That the copyright date is the date when this code is *run* ? As opposed to when this code was written?

Reply:
It's a display thing which I thought was wrong to show an out of date year. It's just for demonstration.